Fluid/Ast/IncludeStatement.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.IO;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Microsoft.Extensions.FileProviders;
+
+namespace Fluid.Ast
+{
+    public class IncludeStatement : Statement
+    {
+        private static readonly IFileProvider _blankFileProvider = new NullFileProvider();
+
+        public IncludeStatement(Expression path)
+        {
+            Path = path;
+        }
+
+        public Expression Path { get; }
+
+        public override async Task<Completion> WriteToAsync(TextWriter writer, TextEncoder encoder, TemplateContext context)
+        {
+            var relativeLayoutPath = (await Path.EvaluateAsync(context)).ToStringValue();
+            var currentViewPath = context.AmbientValues["ViewPath"] as string;
+            var currentDirectory = System.IO.Path.GetDirectoryName(currentViewPath);
+            var partialPath = System.IO.Path.Combine(currentDirectory, relativeLayoutPath);
+            var fileProvider = context.AmbientValues["FileProvider"] as IFileProvider ?? _blankFileProvider;
```

Comment:
Think multi-tenancy, where themes will be looking for partials in different folders